docs/contribute/addnode.qmd

```diff
@@ -16,6 +16,14 @@ struct NewNodeType <: AbstractParameterNode
     # Other fields
 end
 ```
+Another abstract class inherited from `AbstractParameterNode` is called `AbstractDemandNode`. For creating new node type used in allocation, define a struct:
```

Comment:
```suggestion
Another abstract class which inherits from `AbstractParameterNode` is called `AbstractDemandNode`. For creating a new demand node type used in allocation, define a struct:
```

Nice that you thought of also adding this here (: I'm not sure 'inheritance' is the proper word to use here, as that word might be specific for object-oriented languages. Let's ask @visr.

Reply:
Yeah instead of "abstract class inherits", "abstract type subtypes" is perhaps more precise.